src/__snapshots__/SignupModal.test.jsx.snap

```diff
@@ -1,22 +1,22 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`SignupModal should match the snapshot 1`] = `
-<Modal
+<ModalComponent
```

Comment:
these display names are going to get real noisy real fast - I hate to make you go through this again, but it would be super helpful to manually assign the displayName, e.g.

```js
const Modal = withLoading(ModalComponent);
Modal.displayName = 'Modal';
export default Modal;

Reply:
Ahh, I like that. Worth it :)